File: datasets/python/normal/good_code.py
def calculate_sum(numbers):
    """
    Calculate the sum of a list of numbers.
    """
    total = 0
    for number in numbers:
        total += number
    return total

if __name__ == "__main__":
    numbers = [1, 2, 3, 4, 5]
    result = calculate_sum(numbers)
    print(f"Sum: {result}")

File: datasets/java/normal/GoodCode.java
/**
 * Calculate the sum of an array of numbers.
 */
public class GoodCode {
    public static int calculateSum(int[] numbers) {
        int total = 0;
        for (int number : numbers) {
            total += number;
        }
        return total;
    }
    
    public static void main(String[] args) {
        int[] numbers = {1, 2, 3, 4, 5};
        int result = calculateSum(numbers);
        System.out.println("Sum: " + result);
    }
}